packages/vscode-extension/src/debug/depsChecker/backendExtensionsInstall.ts

```diff
@@ -1,89 +1,31 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-// NOTE:
```

Comment:
Do we still need `tools/depsChecker/copyfiles`?

Reply:
deleted